wrappers/IWearLogger/src/IWearLogger.cpp

```diff
@@ -56,12 +70,27 @@ using namespace wearable::wrappers;
 class IWearLogger::impl
 {
 public:
+
+    LoggerLevel loggerLevel;
+    void setLoggerLevel(std::string& str);
+
     bool loadSettingsFromConfig(yarp::os::Searchable& config);
     void checkAndLoadBooleanOption(yarp::os::Property& prop,
                                    const std::string& optionName,
                                    bool& option);
+
+    bool configureMatlabBufferManager(const std::string& sensorName, const size_t& channelSize);
+    bool configureYarpBufferManager(const std::string& sensorName);
     bool configureBufferManager();
 
+    inline void prepareYarpBottle(yarp::os::Bottle& b, const std::vector<double>& data)
+    {
+        for (auto& e : data)
```

Comment:
```suggestion
        for (const auto& e : data)
```

Reply:
maybe a `b.clear()` is required?

Reply:
also maybe its more intuitive to keep input arguments before output arguments, differentiated with `const`.

Reply:
Done in https://github.com/robotology/wearables/pull/123/commits/afacfdf1c362f7bf87ef28f853f25037a6a3ca8a

> maybe a b.clear() is required?

@prashanthr05 yes it is required, thanks!